.github/workflows/gradle.yml

```diff
@@ -1,35 +1,46 @@
-name: Reposilite CI
+# GitHub Actions workflow to automatically build Reposilite.
+name: "Gradle"
 
 on:
   push:
-    branches: [ master, main, "3.0" ]
+    branches: [ "main", "3.0" ]
   pull_request:
-    branches: [ master, main, "3.0" ]
+    branches: [ "main", "3.0" ]
 
 jobs:
   build:
-    name: "Build with JDK${{ matrix.jdk }}"
+    name: "Build with Java ${{ matrix.java-version }}"
```

Comment:
I think `JDK` was slightly better in this context, because we're testing JDK version & platform compatibility, not language itself (like e.g. target lang version etc.)